src/formatDate.js

```diff
@@ -50,7 +50,59 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const firstDivider = fromFormat[3];
+  const secondDivider = toFormat[3];
+  const originalDate = date.split(firstDivider);
+
+  const reverseDate = [];
+  const dateObj = {
+
+  };
+
+  for (let i = 0; i < 3; i++) {
+    switch (fromFormat[i]) {
+      case 'DD':
+        dateObj.DD = originalDate[i];
+        break;
+
+      case 'MM':
+        dateObj.MM = originalDate[i];
+        break;
+
+      case 'YYYY':
+        dateObj.YYYY = originalDate[i];
+        break;
```

Comment:
```suggestion
  dateObj[fromFormat[i]] = originalDate[i];
```
smth like that instead of all cases